src/main/kotlin/com/example/noswimmingserver/domain/common_user/presentation/dto/response/TokenResponse.kt

```diff
@@ -0,0 +1,8 @@
+package com.example.noswimmingserver.domain.common_user.presentation.dto.response
```

Comment:
🚫 **[ktlint]** <package-name> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Package name must not contain underscore (cannot be auto-corrected)